dvc/command/completion.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import logging
+import os
+
+import shtab
+
+from dvc.command.base import CmdBaseNoRepo, append_doc_link
+
+logger = logging.getLogger(__name__)
+DEFAULT_FILENAME = {"bash": "dvc", "zsh": "_dvc"}
```

Comment:
Would it make sense to use `_dvc` for bash as well?

Reply:
doesn't seem standard for bash

Reply:
in any case we're removing this